src/shared/components/oncoprint/Oncoprint.tsx

```diff
@@ -335,27 +335,27 @@ export default class Oncoprint extends React.Component<IOncoprintProps, {}> {
 
     private refreshOncoprint(props: IOncoprintProps) {
         const now = performance.now();
-        if (!this.oncoprint) {
+        if (!this.oncoprintJs) {
             // instantiate new one
-            this.oncoprint = new OncoprintJS(
+            this.oncoprintJs = new OncoprintJS(
                 `#${props.divId}`,
                 props.width,
                 props.initParams
             );
-            this.oncoprint.setTrackGroupLegendOrder([
+            this.oncoprintJs.setTrackGroupLegendOrder([
                 GENETIC_TRACK_GROUP_INDEX,
                 CLINICAL_TRACK_GROUP_INDEX,
             ]);
-            (window as any).frontendOnc = this.oncoprint;
-            if (props.oncoprintRef) {
-                props.oncoprintRef(this.oncoprint);
+            (window as any).frontendOnc = this.oncoprintJs;
+            if (props.oncoprintJsRef) {
+                props.oncoprintJsRef(this.oncoprintJs);
```

Comment:
maybe we should rename this ```broadcastOncoprintJsRef``` ?

Reply:
Fixed in [0349435](https://github.com/cBioPortal/cbioportal-frontend/pull/4239/commits/0349435fbce35c880744526443a91e3aae84d3e9)